main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       dog
+//  action 1:   the dog barks
+dog.bark();
+//  action 2:   the dog sits
+dog.sit();
+//  action 3:   the dog rolls over
+ dog.rollOver();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       airplane
+//  action 1:   the airplane takes off
+airplane.takeOff();
+//  action 2:   the airplane decelerates
+airplane.decelerate;
+//  action 3:   the airplane lands
+ airplane.land();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       fish
+//  action 1:   the fish swims
+fish.swim();
+//  action 2:   the fish eats
+fish.eat();
+//  action 3:   the fish hides
+fish.hide();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
- 
+//  Noun:       cloud
+//  action 1:   the cloud forms
+cloud.form();
+//  action 2:   the cloud rains
+cloud.rain();
+//  action 3:   the cloud dissipates
+cloud.dissipate();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Reply:
Reread instructions, hopefully corrected.